docs/2-configuration.md

```diff
@@ -52,6 +52,8 @@ Here's what you can configure:
 | `WG_DNS_ENABLED`           | `--[no-]dns-enabled`       | `dns.enabled`          |          | `true`                                       | Enable/disable the embedded DNS proxy server. This is enabled by default and allows VPN clients to avoid DNS leaks by sending all DNS requests to wg-access-server itself.         |
 | `WG_DNS_UPSTREAM`          | `--dns-upstream`           | `dns.upstream`         |          | _resolvconf autodetection or Cloudflare DNS_ | The upstream DNS servers to proxy DNS requests to. By default the host machine's resolveconf configuration is used to find its upstream DNS server, with a fallback to Cloudflare. |
 | `WG_DNS_DOMAIN`            | `--dns-domain`             | `dns.domain`           |          |                                              | A domain to serve configured devices authoritatively. Queries for names in the format <device>.<user>.<domain> will be answered with the device's IP addresses.                    |
+| `WG_CLIENTCONFIG_DNS_SERVERS`       | `--clientconfig-dns-servers`       | `clientconfig.dnsservers`      |      |                         | DNS servers to write into the client configuration file. Are used instead of the servers DNS settings, if set.                                                                     |
```

Comment:
Maybe we should mention that it should be a "comma-seperated list" here?

Reply:
There are already directions for multiple values above the table:

> The format for specifying multiple values for options that allow it is:
> * as commandline flags:
>   * repeat the flag (e.g. `--dns-upstream 2001:db8::1 --dns-upstream 192.0.2.1`)
>   * separate the values with a comma (e.g. `--dns-upstream 2001:db8::1,192.0.2.1`)
> * as environment variables:
>   * separate with a comma (e.g. `WG_DNS_UPSTREAM="2001:db8::1,192.0.2.1"`)
>   * separate with a new line char (e.g. `WG_DNS_UPSTREAM=$'2001:db8::1\n192.0.2.1'`)
> * in the config file as YAML list.

I can add a hint that the item accepts one or more values.